tf_bridge.launch.py demo on ros2 branch tries to launch 'ign gazebo'

<!-- If you're not sure whether your problem is a bug, please ask a question at
http://answers.gazebosim.org instead.-->

## Environment
* OS Version: Ubuntu 22.04
* Source or binary build? source, ros2 branch

## Description
* Expected behavior: <!-- Tell us what you expected to happen -->

Demo `ros2 launch ros_gz_sim_demos tf_bridge.launch.py` should work

* Actual behavior: <!-- What happened instead -->

tf_bridge.launch.py tries to launch 'ign gazebo' instead of 'gz sim'

## Steps to reproduce
<!-- Provide steps so we can try to reproduce this issue -->

1. install Gazebo Garden
2. build ros_gz from source on on the ros2 branch
3. ros2 launch ros_gz_sim_demos tf_bridge.launch.py

## Output

[ERROR] [ign-1]: process has died [pid 565049, exit code 255, cmd 'ign gazebo -r /home/clyde/colcon_ws/install/ros_gz_sim_demos/share/ros_gz_sim_demos/models/double_pendulum_model.sdf'].

--- a/ros_gz_sim_demos/launch/tf_bridge.launch.py
+++ b/ros_gz_sim_demos/launch/tf_bridge.launch.py
@@ -27,7 +27,7 @@ def generate_launch_description():
         # Launch gazebo
         ExecuteProcess(
             cmd=[
-                'ign', 'gazebo', '-r',
+                'gz', 'sim', '-r',
                 os.path.join(
                     pkg_ros_gz_sim_demos,
                     'models',

